### ISSUE TITLE
Diagram Simplification

### ISSUE THREAD

**Issue by goatwasher** [InsideTheSim/ethmerge.com-content#37]
I think that the diagram can be simplified even more to make it as easy as possible for new users. Furthermore, the color scheme of the page (the color gradient) could also be included in the diagram and thus be used communicatively as a sign for the change PoW / PoS.

See example of simplified diagram below.

![C3Phmdz](https://user-images.githubusercontent.com/82184504/114110414-bbbc4300-98d7-11eb-81a7-e8dd9a572077.png)

**Comment by yulesa** [comment]
I've liked it.
I've tried to improve it by adding the data layer, because most of question I receive about ETH2 is about how dApp and wallets will work on it. Don't know if it fits your minimalist design.
![image](https://user-images.githubusercontent.com/39807934/114112156-2c506900-98b2-11eb-8aa7-3b7816b92b03.png)

**Comment by goatwasher** [comment]
Quick draft how this *could* work. Its visually a bit harder to grasp of course.

<img width="1045" alt="Diagram Example" src="https://user-images.githubusercontent.com/82184504/114113886-fa55fb80-98df-11eb-99af-d9e61582dd2c.png">

**Comment by InsideTheSim** [comment]
Hey @goatwasher great to see you here! 

While in general I am a fan of the style treatment there's a few things that I'm hesitant about with this one for use in the hero:
- There's been discussion in #33 about ensuring that the diagram is both easy to understand but also that it indicates clearly that PoW will no longer be a part of the chain. The original graphic (before the current one) made both appear co-equal.
- The layout in the header actually benefits really well from the shape of the current graphic
- I like the toned down (grayscale) treatment of the current hero graphic because it keeps the accent focus on the copy.

That said, I'd love to start normalizing more than _just text_ in the FAQs and a more streamlined and less vertically tall asset like the one you've created could work well as an inline image inside one of the FAQs such as: https://ethmerge.com/?q=what-is-the-merge or https://ethmerge.com/?q=can-i-stay-on-ethereum-10-after-the-merge

Thoughts?

**Comment by goatwasher** [comment]
Understood. I would recommend staying in a consistent style page-wide. For example, if the hero stays as it is, inline images further down the page should also be in that style. Different styles may lead to confusion for the user.

**Comment by InsideTheSim** [comment]
I can appreciate that sentiment - but given that the voice is already going to differ (due to different authors) I'm not super worried about having a singular style for artwork. Some diagrams can be more flashy, others more technical. Happy to try and integrate your work if you'd like to discuss it. Getting image assets supported is going to required some thinking on my end due to the 2-repo nature of this website setup.

**Comment by goatwasher** [comment]
Yeah sure, hit me up when you need something. I can provide SVGs.

**Comment by yulesa** [comment]
> I don't indicates clearly that PoW will no longer be a part of the chain.

In the current design there is a X that I don't know if it's a X or a + in the direction of the arrow.

**Comment by Myu-Unix** [comment]
Just a side note it's best to use Ethereum or Eth as the chain name and not ETH (the coin)
